src/services/SubmissionService.js

```diff
@@ -310,10 +310,16 @@ function * createSubmission (authUser, files, entity) {
   if (_.intersection(authUser.roles, ['Administrator', 'administrator']).length === 0 && !authUser.scopes) {
```

Comment:
should the create/update scopes be checked here? it's possible, for instance, to have a read-only M2M token

Reply:
It is being checked at the route level - https://github.com/topcoder-platform/submissions-api/blob/develop/src/routes/SubmissionRoutes.js#L12

Here, we are only checking if it is a m2m token (in which case, the IF condition fails)